app/helpers/search_helper.rb

```diff
@@ -31,9 +31,9 @@ def results_query_label(query)
     "for: #{results.join(' | ')}"
   end
 
-  def sort_by_link(title, sort_key, query, record_type)
+  def sort_by_link(title, sort_key, query, record_type, specified_url: false)
```

Comment:
The changes to this function do not require additional changes in the original use case, but should make it more broadly available for future use (as it can easily drive sorting on all the draft derivative index pages).

Reply:
1. inconsequential, but I think `nil` makes more sense for specified_url
2. more important, I think you should either make the default value `specified_url = false` (or nil), or convert the whole method to using keyword arguments. I don't think we should mix them.